pyInterface/testFileManager.py

```diff
@@ -31,14 +29,14 @@
 	#print f.areDataFilesSynced()
 	#print f.areKeyFilesSynced()
 
-	for binID in f.getBinIDList():
-		for keyFileID in f.getKeyFileIDList():
-			for eventsType in [ pyRootPwa.core.eventMetadata.REAL, pyRootPwa.core.eventMetadata.GENERATED, pyRootPwa.core.eventMetadata.ACCEPTED ]:
-				dataFile = f.getDataFile(binID, eventsType)
-				keyFile = f.getKeyFile(keyFileID)
-				if not dataFile:
-					continue
-				print str(binID) + "	" + str(keyFileID) + "	" + f.getAmplitudeFilePath(binID, keyFileID, eventsType)
-				#print "submit job: calcAmplitudes(" + keyFile.keyFileName + ", " + dataFile.dataFileName + ", -o " + f.getAmplitudeFilePath(binID, keyFileID, eventsType) + ")"
-	print ">> found " + str(len(f.getBinIDList())) + " binIDs and " + str(len(f.getKeyFilePaths())) + " keyFiles."
-	print ">> submitted " + str(len(f.getBinIDList()) * len(f.getKeyFilePaths()) * len(f.dataFiles)) + " jobs."
+# 	for binID in f.getBinIDList():
+# 		for keyFileID in f.getKeyFileIDList():
+# 			for eventsType in [ pyRootPwa.core.eventMetadata.REAL, pyRootPwa.core.eventMetadata.GENERATED, pyRootPwa.core.eventMetadata.ACCEPTED ]:
+# 				dataFile = f.getDataFile(binID, eventsType)
+# 				keyFile = f.getKeyFile(keyFileID)
+# 				if not dataFile:
+# 					continue
+# 				print str(binID) + "	" + str(keyFileID) + "	" + f.getAmplitudeFilePath(binID, keyFileID, eventsType)
+# 				#print "submit job: calcAmplitudes(" + keyFile.keyFileName + ", " + dataFile.dataFileName + ", -o " + f.getAmplitudeFilePath(binID, keyFileID, eventsType) + ")"
+# 	print ">> found " + str(len(f.getBinIDList())) + " binIDs and " + str(len(f.getKeyFilePaths())) + " keyFiles."
+# 	print ">> submitted " + str(len(f.getBinIDList()) * len(f.getKeyFilePaths()) * len(f.dataFiles)) + " jobs."
```

Comment:
Why is this being commented? If it needs to stay like this, could you please remove the spaces after the `#`?